composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "mantisbt/mantisbt",
+    "description": "Mantis Bug Tracker",
+    "type": "project",
+    "require": {
+        "slim/slim": "^3.0"
+    },
+    "license": "GPL v2",
+    "authors": [
+        {
+            "name": "Victor Boctor",
+            "email": "vboctor@gmail.com"
```

Comment:
This was probably picked up from some machine configs.  I don't recall setting it.  Anyways, I'm already using these same MantisBT team details in headers for all files.